charter.md

```diff
@@ -72,28 +78,156 @@ this works.
 
 ### Vacancies
 
-In the event of a resignation or other loss of an elected steering committee
-member, the candidate with the next most votes from the previous election will
-be offered the seat.  This process will continue until the seat is filled.
+In the event of a resignation or other loss of an elected committee member, the
+next most preferred candidate from the previous election will be offered the
+seat.
+
+A maximum of one (1) committee member may be selected this way between
+elections.
 
 In case this fails to fill the seat, a special election for that position will
-be held as soon as possible. [Eligible voters][voter-eligibility] from the most
-recent election will vote in the special election i.e., eligibility will not be
-redetermined at the time of the special election. A committee member elected in
-a special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder.
+be held as soon as possible.
+
+Examples (non-exhaustive):
+
+* the next most preferred candidate is no longer interested in sitting on the
```

Comment:
currently ... we go down the list.

Reply:
as pointed out in some other comment, this could easily choose the community's least preferred candidate if the field isn't large some year.

Reply:
> This does not say how far down the list we go. If the first non-elected person declines, do we consider the second, third, ... tenth? Or is it only the first runner up who is offered a seat?

@thockin -- Here's some of what was changed in this section:

> This process will continue until the seat is filled.

was removed.

> A maximum of one (1) committee member may be selected this way between
elections.

was added. (It's a few lines up on L85-L86.)

That's meant to say:
Only _the next preferred candidate_ from the previous election will be considered.
If for some reason they can't accept, then we hold a special election.

---

> as pointed out in some other comment, this could easily choose the community's least preferred candidate if the field isn't large some year.

The comment @lavalamp is referencing @jberkus in https://github.com/kubernetes/steering/pull/248#issuecomment-1179276881:

> > 2 down the stack feels arbitrary and doesn't account for the % of votes or skew between one or both of the next 2 and the 3-4 members seated from the election. If you took the top 4 and the skew between the 5th and the 4th is small but the skew between the 4/5th and the 6th is huge... it doesn't feel very representative of the community's will. It makes more sense to go down the stack so long as the candidates are within X% of the last seated candidate otherwise hold a special election.
> > I think these two are potentially sub-optimal outcomes: 1.2..3..4.5.................6 or 1..2.3.4..........5...........6
> 
> I share Kirsten's reservations here. It really depends on the election, and that's hard to incorporate into the rules. For example, last steering election we had 13 candidates, so it's reasonable to assume that anyone in the top 7 is fairly well "liked" by the contributors. BUT ... we've had one steering election where there were 5 candidates for 3 seats, and 2 down means picking the candidate who was the least preferred. If you combine that with the open nature of our elections -- where anyone can run, even if they have no prior association with Kubernetes -- it's a recipe for worry.
> 
> So I'd be more comfortable if using alternates from the prior election ended at 1. That is, the seat is offered to the next-most-preferred candidate. If they refuse it, we have a special election.

Reply:
I think the issue is that "selected" implies asking the person and going down the list until someone says "yes". If you change to "selection attempt" or "at most one person shall be invited this way" then it won't be ambiguous.

Reply:
> suggestion: drop the examples section, as the actual text covers our intent

Done. Resolving convo.